cmd/resource-topology-exporter/main.go

```diff
@@ -63,7 +64,8 @@ const helpTemplate string = `{{.ProgramName}}
                                   [Default: /kubeletstate/config.yaml]
   --kubelet-state-dir=<path>...   Kubelet state directory (RO access needed), for smart polling.
   --podresources-socket=<path>    Pod Resource Socket path to use.
-                                  [Default: /podresources/kubelet.sock]`
+                                  [Default: /podresources/kubelet.sock]
+  --reference-container=<spec>    Reference container (namespace/podname/containername)`
```

Comment:
We should add a better description for the user here. Since there are no default values here it might appear that this is an optional flag which is not the case as we always need to determine the shared pool. So lets mention explicitly that if values are not provided here, environment variables will be used and we should have some default values in hack/get-manifest-ds.sh.

Reply:
good point, will add.